crypto.go

```diff
@@ -646,10 +684,10 @@ const (
 )
 
 var kems = map[KEMID]KEMScheme{
-	DHKEM_X25519: dhkemScheme{x25519Scheme{}},
-	DHKEM_X448:   dhkemScheme{x448Scheme{}},
-	DHKEM_P256:   dhkemScheme{ecdhScheme{curve: elliptic.P256()}},
-	DHKEM_P521:   dhkemScheme{ecdhScheme{curve: elliptic.P521()}},
+	DHKEM_X25519: dhkemScheme{group: x25519Scheme{}, skE: nil, pkE: nil},
```

Comment:
No need to specify those fields; by default they're set to their zero value, which is nil.

Reply:
Oh, cool. I'll remove them!